src/components/AppLayout/Header/components/Notifications/index.tsx

```diff
@@ -0,0 +1,196 @@
+import { ReactElement, useMemo, useRef, useState } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { IconButton, Badge, ClickAwayListener, Paper, Popper } from '@material-ui/core'
+import NotificationsNoneIcon from '@material-ui/icons/NotificationsNone'
+import styled from 'styled-components'
+import { orderBy } from 'lodash'
+
+import { ReturnValue as Props } from 'src/logic/hooks/useStateHandler'
+import {
+  deleteAllNotifications,
+  readNotification,
+  selectNotifications,
+} from 'src/logic/notifications/store/notifications'
+import { black300, border, primary200, primary400, sm } from 'src/theme/variables'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import ExpandLessIcon from '@material-ui/icons/ExpandLess'
+import NotificationList from 'src/components/AppLayout/Header/components/Notifications/NotificationList'
+
+export const NOTIFICATION_LIMIT = 4
+
+// Props will be used in popper
+const Notifications = ({ open, toggle, clickAway }: Props): ReactElement => {
+  const dispatch = useDispatch()
+  const notificationsRef = useRef<HTMLButtonElement>(null)
+  const [showAll, setShowAll] = useState<boolean>(false)
+
+  const notifications = useSelector(selectNotifications)
+  const sortedNotifications = useMemo(() => orderBy(notifications, ['timestamp', 'read', 'action']), [notifications])
```

Comment:
This does not work as intended. I'm looking into this now.

Reply:
I've reverted to the sorting logic with the aforementioned suggestions and it now works as intended.